dkpro-core-corenlp-gpl/src/main/java/de/tudarmstadt/ukp/dkpro/core/corenlp/CoreNlpSegmenter.java

```diff
@@ -170,9 +196,17 @@ protected void process(JCas aJCas, String aText, int aZoneBegin)
             tokenizerAnnotator.getResource().annotate(document);
 
             for (CoreLabel token : document.get(CoreAnnotations.TokensAnnotation.class)) {
-                createToken(aJCas, 
-                        token.get(CharacterOffsetBeginAnnotation.class) + aZoneBegin,
-                        token.get(CharacterOffsetEndAnnotation.class) + aZoneBegin);
+                //useCoreLabelWord to be set to true when allowing clitics in the language
```

Comment:
Is the "useCoreLabelWord" a preparation for some later feature?

Reply:
Its already being used. When clitics are splitted it is used to add the whole word as the form of a token instead of the area covered in the text, e.g. (al -> a el) instead of (al -> a l)